app/build.gradle

```diff
@@ -2,13 +2,14 @@ apply plugin: 'com.android.application'
 apply from: "$project.rootDir/config/quality.gradle"
 
 project.ext {
-    supportLibraryVersion = "25.4.0"
+    supportLibraryVersion = "26.1.0"
     daggerVersion = "2.11"
     butterKnifeVersion = "8.7.0"
     rxJavaVersion = "2.1.0"
     rxAndroidVersion = "2.0.1"
     timberVersion = "4.5.1"
-    lifecycleVersion = "1.0.0-alpha4"
+    roomVersion = "1.0.0-rc1"
+    lifecycleVersion = "1.0.0-rc1"
```

Comment:
@jshvarts Question here: What do you think of consolidating these two lines, e.g. `archVersion` or `androidArchVersion` - since using different versions for room and lifecycle would make little sense?